web/docs/guides/integrations/directus.mdx

```diff
@@ -120,11 +121,9 @@ Below are some additional resources to dive in and start exploring these two pla
 **Directus**
 - See the [Directus guides](https://directus.io/guides/).
 - Join the Directus community on [Discord](https://directus.chat/).
-- Check out the source code on the official [Directus GitHub Repo](https://github.com/directus/directus).
+- Check out the source code and give us a star on the official [Directus GitHub Repo](https://github.com/directus/directus).
```

Comment:
```suggestion
- Check out the source code on the official [Directus GitHub Repo](https://github.com/directus/directus).
```

Removing "give us a star" - seems promotional, and also this is in the supabase docs so "us" is Supabase, not directus